script/check-dotnet-format.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# shellcheck source=../lib/setup.sh
+source "$(dirname "${BASH_SOURCE[0]}")/../lib/setup.sh" || exit "${EXIT_FAILED_TO_SOURCE}"
+
+# It's legitimate for a linter to exit non-zero; some indicate success-but-lint that way.
+set +o errexit
+
+# Skip bash stack-trace, otherwise our bash stack-traces clutter up the output.
+export SKIP_BASH_STACKTRACE=1
+
+exec dotnet-format --check . --folder
```

Comment:
Does that `.` conflict with the args passed in by `pre-commit`? As above, it would be nicer to be able to pass just the filenames, since then
* it'll be more efficient
* `pre-commit`'s incremental linting can work, which is more efficient still

Reply:
Please refer to above comment (happy to make the change if preferable, but initially was not working)